os/fs/driver/mtd/smart.c

```diff
@@ -2163,6 +2163,44 @@ static int smart_scan(FAR struct smart_struct_s *dev)
 #endif
 			}
 
+			fdbg("Duplicate Sector winner=%d, loser=%d\n", winner, loser);
+
+#ifdef CONFIG_MTD_SMART_ENABLE_CRC
+			/* Check CRC of the winner sector just in case */
+
+			ret = MTD_BREAD(dev->mtd, winner * dev->mtdBlksPerSector, dev->mtdBlksPerSector, (FAR uint8_t *)dev->rwbuffer);
```

Comment:
I think the header of the sector has already been read at line 2126, so maybe we can read just the remaining part of the sector here?

Reply:
No, winner can be changed based on sequence number, so here we should read winner's data again.